apps/map-editor/src/index.scss

```
@@ -0,0 +1,108 @@
+:root {
+  font-family: "Roboto", sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)